src/app/core/public/api/Circuit.ts

```diff
@@ -45,6 +45,15 @@ export interface Circuit {
     getObjs(): Obj[];
     getComponentInfo(kind: string): ComponentInfo | undefined;
 
+    /**
+     * Returns the average of the positions of the components selected
+     * as a Vector object.
+     * 
+     * @param {String} space defines the coordinate-space which can be 
```

Comment:
You dont need the type for tsdoc comments since the type is already defined in the code